sjb/config/test_cases/test_branch_origin_web_console_server_e2e.yml

```diff
@@ -1,18 +1,18 @@
 ---
-# TODO for now we are just running build-images so we don't need to depend on a built origin AMI
-# but eventually we will need to change this back to do more interesting e2e tests
-# parent: 'common/test_cases/origin_built_release.yml'
-parent: 'common/test_cases/origin.yml'
+parent: 'common/test_cases/origin_built_release.yml'
 extensions:
   sync_repos:
     - name: "origin-web-console-server"
   actions:
     - type: "script"
-      title: "build the web console container image"
+      title: "build and run the web console container image with cluster up"
       repository: "origin-web-console-server"
       timeout: 3600
+      # TODO For now, run build-images and test cluster up to verify the console starts,
+      # but eventually we will want to do more interesting e2e tests.
       script: |-
+        sudo yum --disablerepo=\* --enablerepo=origin-local-release,oso-rhui-rhel-server-releases install -y origin-clients
         make build-images
-    # TODO deploy the image and validate it serves content
+        oc cluster up --version=latest --public-hostname=localhost --loglevel=5
```

Comment:
if you make this a separate item in the actions array, then it can have its own title, and the roll up of the error will be better